src/scrabble.js

```diff
@@ -1,5 +1,82 @@
-function scrabble() {
-
+function scrabble(str) {
+  let counter = 0 // must be local to restart and redefine back to 0 when finished otherwise global scope will add it up
+  if (!str || str.length === 0) {
+    return 0
+  }
+  for (let i = 0; i < str.length; i++) {
```

Comment:
good use of if statement in the for loop and assigning each index value of string.